src/Lachain.Benchmark/BlockchainBenchmark.cs

```diff
@@ -387,6 +399,51 @@ EcdsaKeyPair keyPair
             Console.WriteLine(
                 $"Balance After Transaction {_stateManager.LastApprovedSnapshot.Balances.GetBalance(keyPair.PublicKey.GetAddress())}");
         }
+        
+        private void _Bench_Tx_Pool(
+            ITransactionBuilder transactionBuilder,
+            ITransactionSigner transactionSigner,
+            EcdsaKeyPair keyPair)
+        {
+            const int txGenerate = 1000;
+            const int txPerBlock = 1000;
+
+            Logger.LogInformation($"Setting initial balance for the 'From' address");
+            _stateManager.LastApprovedSnapshot.Balances.AddBalance(keyPair.PublicKey.GetAddress(),
+                Money.Parse("200000"));
+            
+            ITransactionPool transactionPool = _container.Resolve<ITransactionPool>();
+            
+            var watch = System.Diagnostics.Stopwatch.StartNew();
+            for (int i = 0; i < txGenerate; i++)
+            {
+                var randomValue = new Random().Next(1, 100);
+                var amount = Money.Parse($"{randomValue}.0").ToUInt256();
+
+                byte[] random = new byte[32];
+                RNGCryptoServiceProvider rng = new RNGCryptoServiceProvider();
+                rng.GetBytes(random);
+
+                var tx = new Transaction
+                {
+                    To = random.Slice(0, 20).ToUInt160(),
+                    From = keyPair.PublicKey.GetAddress(),
+                    GasPrice = (ulong)Money.Parse("0.0000001").ToWei(),
+                    GasLimit = 100000000,
+                    Nonce = (ulong)i,
+                    Value = amount
+                };
+
+                transactionPool.Add(transactionSigner.Sign(tx, keyPair), false);
```

Comment:
Can we separate transaction creation and adding them  to pool? Tx signing can be time consuming operation and we can't optimize it,  let's don't count it. Create a txes before starting to measure the time,  place them to some array,  and after that measure the time required to add ready txes to the pool.